build/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "build",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "build": "yarn build:ng7",
```

Comment:
What is the advantage now to generate for Angular 7? How we detect that's correct build for Angular 8 in CI?

Reply:
The fact is that if we gen using Angular 8 then it breaks compatibility of the lib to all previous versions of angular because of a TS 3.4 specific change.
Doing the build with Angular 7 ensures that we maintain compatibility backwards and with Angular 8. Please read the description of the PR for context.

Reply:
Ok